### ISSUE TITLE
People app's avatar volume controls don't work properly

### ISSUE THREAD

**Issue by ctrlaltdavid** [vircadia/vircadia#723]
If you try reducing an avatar's volume the volume isn't reduced until after you move the slider all the way to the left - then the volume slider works as expected.

**Comment by JulianGro** [comment]
You mean the persons volume?
I never had that issue, but I do have the issue that the value gets reset and I need to do a change for it to work again.

**Comment by ctrlaltdavid** [comment]
Yes, each person's volume. Similar problem occurs when trying to increase a person's volume.

**Comment by daleglass** [comment]
I have the impression that sometimes it doesn't update on the first click and needs multiple changes to take effect

**Comment by daleglass** [comment]
Dev meeting: settings may be randomly resetting and not being kept if somebody leaves and returns. Possibly the settings should persist?

**Comment by JulianGro** [comment]
I think this existing issue is enough.
